--- origsrc/qtzmanim-1.1/mainform.ui.h	2006-09-01 15:16:40.000000000 -0500
+++ src/qtzmanim-1.1/mainform.ui.h	2008-07-24 19:20:24.218750000 -0500
@@ -28,7 +28,7 @@
 SHKIAH_AZIMUTH=NETZ_AZIMUTH,
 ALOT_AZIMUTH=90+16.1,
 MESHEYAKIR_AZIMUTH=90+11,
-TZAIT_AZIMUTH=90+8.5;
+TZAIT_AZIMUTH=90+6.5;
 double const ph[]={3,4,6,6.5,9.5,10.75}; // loop through these right after updating netz
 
 
--- origsrc/qtzmanim-1.1/mydialog.ui.h	2006-09-01 14:56:11.000000000 -0500
+++ src/qtzmanim-1.1/mydialog.ui.h	2008-07-24 19:20:24.234375000 -0500
@@ -28,7 +28,7 @@
 {
     QString text;
     text+="This program was written by Ken Bloom (kbloom@gmail.com)\n\n";
-    text+="Nightfall (8.5 solar degrees)\n";
+    text+="Nightfall (6.5 solar degrees)\n";
     text+="Daybreak (16.1 solar degrees)\n";
     text+="Earliest Talit (11 solar degrees)\n";
     text+="All ofther times according to GR\"A\n";
